src/constants.ts

```diff
@@ -1,10 +1,17 @@
 import path from 'path';
 
-export const DEFAULT_EXPIRATION = '14d';
+export const DEFAULT_EXPIRATION = 14;
```

Comment:
without the `d` suffix, do you think `_DAYS_` should be a part of the const name, like the one below?